contracts/RewardManager.sol

```diff
@@ -54,6 +54,7 @@ contract RewardManager is Ownable, Versionable, Safe {
   uint256 public rewardeeRegistrationFeeInSPEND;
   uint256 public rewardProgramRegistrationFeeInSPEND;
   address payable public rewardFeeReceiver; // will receive receive all fees
+  address public governanceAdmin; // eoa with governance powers
```

Comment:
Is the reward manager contract deployed yet? If so, I think adding this here will break storage layout when the implementation is upgraded.

Reply:
It will not be upgrade safe, but we are still in the mode of making destructive changes to the rewards contracts since we have no real clients yet.

Reply:
When @tintinthong is ready for the SDK updates, the subgraph will be updated to start indexing this contract  at the block the new reward manager is deployed

Reply:
there are quite a few changes happening at the same time. I will bottle up all those in a single sdk update.